apps/lend/src/store/createLoanRepaySlice.ts

```diff
@@ -22,6 +22,10 @@ type SliceState = {
   formValues: FormValues
 }
 
+enum FormError {
```

Comment:
You already have `ALERT_FORM_ERROR_KEYS`. Refactor this object to enum and import it as a type